src/context/init.js

```diff
@@ -75,6 +90,7 @@ function initContext() {
   initValue(context);
   initUtils(context);
   initServices(context);
+  initExternals(context);
```

Comment:
Does not work, since `context.addClass(ApimapSender);` will require `superagentRequest` to already be in the context (`Cannot read property 'post' of undefined` when linking this in a lumber project). You should move this `initExternals(context);` before `initValue`